mail_outbound_static/tests/test_ir_mail_server.py

```diff
@@ -0,0 +1,369 @@
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import logging
+import os
+from email import message_from_string
+
+import odoo.tools as tools
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+from odoo.addons.base.tests.common import MockSmtplibCase
```

Comment:
@simahawk @rousseldenis 
With this improvement https://github.com/odoo/odoo/commit/1b9dd118cb0f305ed38b6326fa814336cbf16252
I propose to improve tests, as they are failing using https://docs.python.org/3/library/unittest.mock.html#module-unittest.mock